dev_guide/ssh_environment.adoc

```diff
@@ -0,0 +1,55 @@
+= Opening a Remote Shell to Containers
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+The `oc rsh` command allows you to locally access and manage tools that are on
+the system. The secure shell (SSH) is the underlying technology and industry
+standard that provides a secure connection to the application. Access to
+applications with the shell environment is protected and restricted with
+Security-Enhanced Linux (SELinux) policies.
+
+[[ssh-environment-start-a-secure-shell-session]]
+== Start a Secure Shell Session
+
+Open a remote shell session to a container:
+----
+$ oc rsh <pod_ID>
+----
+
+While in the remote shell, you can issue commands as if you are inside the
+container and perform local operations like monitoring, debugging, and using CLI
+commands specific to what is running in the container.
+
+For example, in a MySQL container, you can use `SELECT COUNT([column]) FROM
+[table];` to count the number of records in the database. You can also use use
```

Comment:
That `SELECT` command is pretty cool.  Unfortunately, it's not a command that you can type directly into the shell; the actual shell command is called `mysql`.  You invoke that command to get another prompt, and then type in the `SELECT` command at that prompt.  Poking around, i see that we already have an example documented -- see `using_images/db_images/mysql.adoc` around line 106.  Perhaps you can marry this `SELECT` command w/ that example (copying it here).  Another idea is to link to it.